src/posts/2021-06-07-fixing-contrast-issues-on-your-own-site-and-elsewhere.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Fixing contrast issues, on your own site and elsewhere
+description: "Insufficient text contrast is the most common accessibility issue on websites today, even though by and large be automatically found and fixed. The Fix Contrast browser extension fixes contrast issues for end users."
+category: Assistive technology
+author: Kilian Valkhof
+date: 2021-06-07
+further_reading:
+  - title: "Fix Contrast website"
+    url: https://fixa11y.com/
+    source: Fix contrast
+  - title: "Fix contrast for Chrome, Vivaldi and Brave"
+    url: https://chrome.google.com/webstore/detail/fix-contrast/pbbcgecjmpkglppfjjggkkbhdnlemhkg
+    source: Fix contrast
+  - title: "Fix contrast for Edge"
+    url: https://microsoftedge.microsoft.com/addons/detail/cglmfaokelifdefnifhfdpmcckjfedeb
+    source: Fix contrast
+  - title: "Polypane"
+    url: https://polypane.app/
+    source: Polypane
+  - title: "What is color contrast"
+    url: https://www.a11yproject.com/posts/2015-01-05-what-is-color-contrast/
+    source: The A11Y Project
+eleventyNavigation:
+  key: { { category } }
+  title: { { title } }
+tags:
+  - wcag
+  - contrast
```

Comment:
```suggestion
  - assistivetechnology
```

This will also help with the post getting displayed. Tag metadata is used for showing content, not what the content contains, which is admittedly a little awkward 😰